semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -122,6 +144,76 @@ object Main {
         }
       }
     }
+    locally {
+      def synthesizeBuiltin(name: String): s.SymbolInformation = {
```

Comment:
What do you think about synthesizing these on the fly at runtime instead of re-generating scala/Any.class and friends for every entry in each classpath?